### ISSUE TITLE
Authenticated Client Not Retrieving Messages

### ISSUE THREAD

**Issue by Weezy11** [danpaquin/coinbasepro-python#348]
Hello,

When I attempt to connect an authenticated client, none of the authenticated methods seem to be working. All of them are raising the below raw_decode error, which I believe is saying no messages are returning from the initial request. However, all public client methods are working. I've looked along the thread and seen that this was a similar problem in Issue #86. I've attempted with multiple API keys, on both the sandbox and normal coinbase pro.  Could you help direct me to a solution?

Code:
#Create Authenticated Client for sandbox
auth_client = cbpro.AuthenticatedClient(api_key, api_secret, pass_phrase, api_url)
try:
    auth_client.get_time()
except:
    auth_client.get_time()
________________________________________________________________________________________________________
Error: 
line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None

**Comment by noah222** [comment]
Have you given the correct permissions to your api key?

**Comment by Weezy11** [comment]
Yes, I have both view and trade permissions, no transfer permission though.

**Comment by noah222** [comment]
Does it create the exact same error without a URL parameter?

**Comment by Weezy11** [comment]
Thanks, that seemed to fix the issue for connecting to regular coinbase pro, however, do you know how I could connect to the sandbox?

**Comment by noah222** [comment]
Did you create a unique api key for the sandbox first? Just checking the basics. But I've also heard the sandbox support can be unreliable and I have not personally used it in a while.

**Comment by Weezy11** [comment]
I did create a unique api key for the sandbox, but what you say makes sense.
